packages/backend/src/api-impl.ts

```diff
@@ -90,13 +99,18 @@ export class BootcApiImpl implements BootcApi {
     return stopCurrentVM();
   }
 
-  async deleteBuilds(builds: BootcBuildInfo[]): Promise<void> {
+  async deleteBuilds(buildIds: string[]): Promise<void> {
     const response = await podmanDesktopApi.window.showWarningMessage(
       `Are you sure you want to remove the selected disk images from the build history? This will remove the history of the build as well as remove any lingering build containers.`,
       'Yes',
       'No',
     );
     if (response === 'Yes') {
+      // create an array of builds. invalid build ids are ignored
+      const builds = buildIds
+        .map(id => this.history.getHistory().find(build => build.id === id))
+        .filter(build => build) as BootcBuildInfo[];
```

Comment:
issue: since TypeScript 5.5 / https://github.com/microsoft/TypeScript/pull/57465 the inferring is automatic on the filter so no need to add a cast to as `BootcBuildInfo[]`

but use `filter(build => !!build)` rather than `(build => build)`

https://www.typescriptlang.org/play/?ts=5.7.3#code/C4TwDgpgBAsiDyAjAVhAxsKBeKBvAlgCYBcUAdgK4C2iEATgDRTAQAewpAzsHfmQOYBfANwAoUWgD2ZblCoIU6YJ1JwkqDAG0AutiiaCJKAEYmLdqQDkAM0mTLgpodIAmM2w5RLiAIZ0HTkSkpsweVr4AXg7aYhLSsvicMD7AaAAWfPzqSnoAFJIoqgoawACU2AB8eKJQUPjWUPkoAHRE2Fg4xuW4NbVQdBDAFHRkUAXIvYJQEAA2nNA9ff2Dw6MUZIQQ1nwQhGK1U5NQsVIymNszLAOEevLZGJzNVD5gTciVdUkp6Zn3wG+lUpiU6yMjSACqGy2O0IABVZhAqBAyMo9Bcrrtmuj6Ll8CwqB8AISEvGIoFxM5QHwzGYAMUk60IKjUii0uhwYLIkM22zIu3hM0RyOUJ3ikkFzRmkn4uUsYPIEF2zEkVMINw8dB8UDQPm4TAGnAol0+liY1LpDI2nFKQA

Reply:
Nice! I wasn't aware of that, fixed.